recipes/btllib/meta.yaml

```
@@ -39,7 +39,7 @@ requirements:
     - tar
     - bzip2
     - xz
-    - lrzip
+    - lrzip # [linux or osx and x86_64]
```

```suggestion
    - lrzip # [not arm64]
```
This might be a little less confusing since it's the lack of arm64 build that's the problem. In the [Conda docs](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors), it says that only applies to osx (or windows).